tests/integration/questionnaire/test_questionnaire_relationships_unrelated.py

```diff
@@ -0,0 +1,47 @@
+from . import QuestionnaireTestCase
+
+
+class TestQuestionnaireRelationshipsUnrelated(QuestionnaireTestCase):
+    def launch_survey_and_add_people(self):
+        self.launchSurvey("test_relationships_unrelated", roles=["dumper"])
+        self.add_person("Marie", "Doe")
+        self.add_person("John", "Doe")
+        self.post({"anyone-else": "No"})
+
+    def test_relationship_unrelated_is_accessible_when_list_name_and_list_item_valid(
```

Comment:
The test names could be made shorter without losing any clarity by removing `relationship_unrelated` / `unrelated_relationship` (as all of the tests in this class are testing unrelated relationships).